centaur/src/main/resources/standardTestCases/output_filename_interpolation/output_filename_interpolation.wdl

```diff
@@ -8,11 +8,12 @@ task output_filename_interpolation {
         echo a > ${outputMain}
         echo b > ${outputMain + "." + outputSuffix}
         echo c > ${outputPrefix}/${outputMain}.${outputSuffix}
-        echo ${outputMain} > someFile.txt
+        echo foo > bar
+        echo bar > baz
     }
 
     output {
-        File f = read_string("someFile.txt")
+        File f = read_string("baz")
```

Comment:
I don't believe this will delocalize `bar`. So if you try to use `f` in a subsequent task, it won't be available